pkg/adapter/v2/main.go

```diff
@@ -105,7 +107,7 @@ func MainWithContext(ctx context.Context, component string, ector EnvConfigConst
 		logger.Error("Error loading cloudevents overrides", zap.Error(err))
 	}
 
-	eventsClient, err := NewCloudEventsClient(env.GetSink(), ceOverrides, reporter)
+	eventsClient, err := kncloudevents.NewCloudEventsClient(env.GetSink(), ceOverrides, reporter)
```

Comment:
this is incorrect. The client created inside of Adapter was specific to `Adapters` and not general.

Reply:
fixed.